File: freecodecamp/codeInterviewPrep/symmetric_difference.py
def sym(list_one, list_two):
    new_list = []
    
    for item in list_one:
        if item not in list_two and item not in new_list:
            new_list.append(item)
    for item in list_two:
        if item not in list_one and item not in new_list:
            new_list.append(item)
    print(new_list)
    new_list
    return new_list
if __name__ == '__main__':
    sym([1, 2, 3], [5, 2, 1, 4])
    # [3,4,5]
    sym([1, 2, 3, 3], [5, 2, 1, 4])
    # [3, 4, 5]

File: freecodecamp/codeInterviewPrep/symmetricDifference.js
// function sym(listOne, listTwo) {

//     newList = []

//     for (i = 0; i < listOne.length; i++){
//         if (listTwo.includes(listOne[i]) === false && newList.includes(listOne[i])===false)
//             newList.push(listOne[i])
//     }

//     for (i = 0; i < listTwo.length; i++){
//         if(listOne.includes(listTwo[i])=== false && newList.includes(listTwo[i])===false){
//             newList.push(listTwo[i])
//         }
//     }

    
//     newList = newList.sort()
    
//     console.log(newList)
//     return newList
// }

function sym(args) {
    var myArray = []; // Empty array, at first.
for (var i = 0; i < arguments.length; i++) {
    myArray.push(myArray[i])
}} // Now 'args' is an array that holds your arguments.
  
console.log(sym([1, 2, 3, 3], [5, 2, 1, 4]))

// [3, 4, 5]